packages/rspack/src/exports.ts

```diff
@@ -71,6 +71,8 @@ export {
 	cleanupGlobalTrace as experimental_cleanupGlobalTrace
 } from "@rspack/binding";
 
+export { default as EntryOptionPlugin } from "./lib/EntryOptionPlugin";
```

Comment:
EntryOptionsPlugin shouldn't be exported

Reply:
Is there any way to satisfy this needage?

https://github.com/vanilla-extract-css/vanilla-extract/blob/27552826716993d9923d2aa5f30513594ff3c1ee/packages/webpack-plugin/src/compiler.ts#L88-L91

Reply:
Should I porting it into a built-in plugin in Rust.